### PR TITLE
update django to 3.2

### PR DESCRIPTION
**Pull request by eMBee**

### CODE REVIEW

File: src/pretix/base/i18n.py [hunk 1]
```diff
@@ -73,10 +73,11 @@ def __str__(self):
 def get_babel_locale():
     babel_locale = 'en'
     # Babel, and therefore django-phonenumberfield, do not support our custom locales such das de_Informal
-    if localedata.exists(translation.get_language()):
-        babel_locale = translation.get_language()
-    elif localedata.exists(translation.get_language()[:2]):
-        babel_locale = translation.get_language()[:2]
+    if translation.get_language():
```
Comment (hongquan) [inline src/pretix/base/i18n.py:76]: How about using [walrus operator](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions) to write less code:

```py
if lang := translation.get_language():
    if localedata.exists(lang):
        babel_locale = lang
    elif localedata.exists(lang[:2]):
        babel_locale = lang[:2]
```

File: src/pretix/helpers/templatetags/jsonfield.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# replacement for jsonfallback functions copied from 
+# https://github.com/raphaelm/django-jsonfallback
+
+import copy
+from django.db import NotSupportedError
+from django.db.models import Expression, JSONField
+
+def mysql_compile_json_path(key_transforms):
```
Comment (hongquan) [inline src/pretix/helpers/templatetags/jsonfield.py:8]: We are [removing MySQL support](https://github.com/fossasia/eventyay-tickets/issues/29). You can skip MySQL.

Comment (eMBee) [reply src/pretix/helpers/templatetags/jsonfield.py:8]: sure, but it's not removed yet. i like to keep things consistent

**Review by mariobehling — APPROVED** [review]
Based on the feedback I think we can merge this. If you find any issues, please open a new issue on the tracker. Thanks

File: src/pretix/helpers/templatetags/jsonfield.py (hunk 2 above)
Comment (eMBee) [reply src/pretix/helpers/templatetags/jsonfield.py:8]: actually i missed that mysql removal was just merged, sorry.

Comment (eMBee) [reply src/pretix/helpers/templatetags/jsonfield.py:8]: fixed in a new PR

**Pull request merged**